packages/neon-api/src/provider/common.ts

```diff
@@ -61,3 +61,21 @@ export function findGoodNodesFromHeight(
   const threshold = bestHeight - tolerance;
   return sortedNodes.filter((n) => n.height >= threshold);
 }
+
+export interface Entry {
+  txid: string;
+  time: number;
+  block_height: number;
+  asset: string;
+  amount: string;
+  address_to: string;
+  address_from: string;
+}
+
+export interface IAddressAbstract {

```

Comment:
no need to use the `I` convention found in C#. This is also the exposed interface so can we use JS conventions for all the fields (camelCase instead of snake_case).

Reply:
okay